internal/loadbalancers/create.go

```diff
@@ -0,0 +1,93 @@
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package loadbalancers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	lbaas "github.com/equinix/metal-cli/internal/loadbalancers/api/v1"
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) Create() *cobra.Command {
+	var (
+		name       string
+		locationId string
+		portIds    []string
+		providerId string
+		projectID  string
+	)
+
+	// loadbalancerCreateCmd represents the loadbalancerCreate command
+	createLoadBalancerCmd := &cobra.Command{
+		Use:   `create -n <loadbalancer_name> -l <location_id_or_metro> [-p <project_UUID>] [--provider <provider_id>] [--port <port_UUID>]`,
+		Short: "Creates a loadbalancer.",
+		Long:  "Creates a loadbalancer with the specified name.",
+		Example: `  # Creates a new loadbalancer named dev-loadbal in the Dallas metro: 
+  metal loadbalancer create --name dev-loadbal -l da
+  
+  # Creates a new loadbalancer named prod-loadbal in the DC metro:
+  metal loadbalancer create -n prod-loadbal -l dc`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			// handle metro aliases for locations
+			if LBMetros[locationId] != "" {
+				locationId = LBMetros[locationId]
+			}
+
+			req := c.projectService.CreateLoadBalancer(context.Background(), projectID)
+			// opts := lbaas.NewLoadBalancerCreate(name, locationId, portIds, providerId)
+			opts := &lbaas.LoadBalancerCreate{
+				Name:       name,
+				LocationId: locationId,
+				PortIds:    portIds,
+				ProviderId: providerId,
+			}
+			req = req.LoadBalancerCreate(*opts)
+			lb, _, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("Could not create LoadBalancer: %w", err)
+			}
+
+			data := make([][]string, 1)
+
+			data[0] = []string{lb.GetId(), lb.GetMessage(), lb.GetVersion(), strconv.Itoa(int(lb.GetStatus()))}
+			header := []string{"ID", "Message", "Version", "Status"}
+			return c.Out.Output(lb, header, &data)
+		},
+	}
+
+	createLoadBalancerCmd.Flags().StringVarP(&name, "name", "n", "", "Name of the loadbalancer")
+	createLoadBalancerCmd.Flags().StringVarP(&projectID, "project-id", "p", "", "The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.")
+	createLoadBalancerCmd.Flags().StringVarP(&locationId, "location", "l", "", "The location's ID.")
+	createLoadBalancerCmd.Flags().StringVarP(&providerId, "provider", "r", ProviderID, "The provider ID.")
+	createLoadBalancerCmd.Flags().StringSliceVar(&portIds, "port", []string{}, "The port's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PORT_ID environment variable.")
```

Comment:
Is `METAL_PORT_ID` an existing construct? I guess more curious why this one has an environment variable and the others wouldn't if that's not the case.

Reply:
Technically, every parameter can be supplied as an environment variable or a ~/.config/equinix/metal.yaml setting.

We probably don't need to be explicit here about METAL_PORT_ID (I would guess I copied this from the Project subcommand and made minor adjustments to make it fit).

Reply:
Port isn't required in any case and this comment should have been applied to location. Fixing.